package.json

```diff
@@ -71,7 +71,8 @@
     "chai": "^4.3.4",
     "coveralls": "^3.1.0",
     "cross-env": "^4.0.0",
-    "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0",
+    "escope": "^3.6.0",
+    "eslint": "^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8.0.0-0",
```

Comment:
> However, by not doing that, we could actually merge this now, so that tests are at least running on eslint 8.

:+1: I think it should be added to peerDeps after ESLint v8.0.0 is stable.